docs(sem): add secret protection step in different docs (#2631)
scaleway/docs-content: identity-and-access-management/secret-manager/concepts.mdx

tags: secret-manager secret version
---

<Concept>

## Ephemeral policy

> An ephemeral policy is a statement that defines the lifetime of a secret and its versions, using either of the following options:

Author: ```suggestion
An ephemeral policy is a statement that defines the lifetime of a secret's versions, using either of the following options:
```

- **Single access**: allows you to set your secret versions to **expire after one single access**.
- **Time to Live**: allows you to set a time frame of up to one year, during which your secret versions are valid and accessible.

> The ephemeral policy also allows you to decide whether to **disable** or **delete** your secret's versions, once the secret has expired.

Author: ```suggestion
The ephemeral policy also allows you to decide whether to **disable** or **delete** your secret's versions, once they expire.
```

<Message type="important">
  - The ephemeral policy can only be applied to a secret at creation, and **cannot be removed** once applied.

>   - If applied on a secret, all its versions inherit from the policy.

Reviewer: ```suggestion
  - If applied to a secret, all its versions inherit from the policy.
```

</Message>

</Concept>

<Concept>

## Ephemeral secret

> An ephemeral secret is a secret that has a limited lifetime, enforced by an [ephemeral policy](#ephemeral-policy).

Author: ```suggestion
An ephemeral secret is a secret whose versions have a limited lifetime, enforced by an [ephemeral policy](#ephemeral-policy).
```
